src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * @param string $sku
+     * @param int    $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->cartId = $cartId;
```

Comment:
Why not just public properties? It would ease integration on the UI level (allowing for example Symfony Forms to use them as a Form Model).

I know this would lead to objects that don't have all properties initialized, but event now those properties could be initialized badly.

So it's not matter of "valid" data, but just passing data. We don't have structures in PHP, that's why there is need for DTOs. Let's ease up their creation.